frontend_tests/node_tests/timerender.js

```diff
@@ -232,13 +232,13 @@ run_test('last_seen_status_from_date', () => {
     }
 
     assert_same(function (d) { return d.addSeconds(-20); },
-                i18n.t("Last seen just now"));
+                i18n.t("Just now"));
 
     assert_same(function (d) { return d.addMinutes(-1); },
-                i18n.t("Last seen just now"));
+                i18n.t("Just now"));
 
     assert_same(function (d) { return d.addMinutes(-2); },
-                i18n.t("Last seen just now"));
+                i18n.t("Just now"));
 
     assert_same(function (d) { return d.addMinutes(-30); },
                 i18n.t("Last seen 30 minutes ago"));
```

Comment:
I'm leaving this for a future commit just because I wanted to merge the big stack here, but I'm pretty sure the intent here was that we'd get rid of `Last seen` in all of these.  

@rishig can you confirm?

Reply:
yeah, confirming. I believe "Last seen" (or something similar) is the heading, and avoiding it in the text will also simplify i18n.